ti/instrumentation/python/helper.go

```diff
@@ -31,10 +28,15 @@ func getPythonTestsFromPattern(tests []ti.RunnableTest, workspace, testpattern s
 		if path == "" {
 			continue
 		}
-		test := ti.RunnableTest{
-			Class: path,
+		for _, glob := range testGlobs {
```

Comment:
Not a blocker but to make it consistent with other implementations, you can do something like this: https://github.com/harness/lite-engine/pull/97/files#diff-b5fb100ad990fe806225fea86c0c6369c47a2ad49b94728e266daf3437161abdR21

Reply:
I will keep this one for now. Because the parseJavaNode part is copied from addon. And is quite a detour because we create a Node and covert to a test